### ISSUE TITLE
BatchNorm momentum

### ISSUE THREAD

**Issue by Anzoteh96** [svishwa/crowdcount-mcnn#64]
Hello, 

Not too sure if it's an appropriate to raise this, but I felt it strange that the momentum of batch normalization is set to 0 here:

https://github.com/svishwa/crowdcount-mcnn/blob/e04c67f799ff024dbea93b81a079ffdd1b6daa28/src/network.py#L11

According to the [documentation](https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm2d.html) it seems that we use the average value of the first batch ever seen. 

I wonder if this is intended since many guides suggest 0.1, 0.01, etc (but not 0).
